oi/type_graph/AddPadding.cpp

```diff
@@ -15,9 +15,15 @@
  */
 #include "AddPadding.h"
 
+#include <bitset>
```

Comment:
This one isn't used directly in this file at all - only through oi/Features.h

Reply:
Surprised me too, the reasoning it gave is it needs the implementation of `BitsetType::reference` for the result from `EnumBitset::operator[](T v)`. It's possible to make this one go away by adding `// IWYU pragma: export` to the `EnumBitset.h` header, which seems like a good idea.